### ISSUE TITLE
[RFE] Improve and integrate a11y

### ISSUE THREAD

**Issue by joaquimrocha** [kinvolk/nebraska#363]
## Current situation

Nebraska has certain accessibility issues.

## Impact

Users with sight impairment or other sight related special needs may find the use of Nebraska's UI more difficult than it should be.

## Ideal future situation

A11y has been reviewed, issues fixed, and tools for automated testing have been added to the CI.

## Implementation options

Start with these improvements:
- [x] Chrome lighthouse automatic accessibility audit passes on each screen
- [x] eslint-plugin-jsx-a11y passing and integrated into CI
- [x] Keyboard navigation is possible for each screen

## Additional information

Headlamp is undergoing a similar a11y improvement: https://github.com/kinvolk/headlamp/issues/245
